services/blockchain-service/blockchain-service-base.js

```diff
@@ -285,7 +285,7 @@ class BlockchainServiceBase {
                 data: { tokenId },
             });
 
-            return tokenId;
+            return [tokenId, receipt];
```

Comment:
Let's return an object to avoid mistakes in the future
```suggestion
            return { tokenId, receipt };
```